SC_redesign/Crop_and_Soil/tests_SC/crop_tests/test_crop_management.py

```diff
@@ -179,25 +179,31 @@ def test_manage_harvest(harvest_op: HarvestOperation, field_name: str, field_siz
     (0.9, 0.85, False),  # harvest and collect
     (0, 0, True),  # harvest override
     (0.9, 0.85, True),  # harvest override
+    (-1, 0.85, True),
+    (0, 1.5, True)
 ])
-def cut_crop(efficiency: float, harvest: float, override: bool):
+def test_cut_crop(efficiency: float, harvest: float, override: bool):
     """ensure that the crop cutting routines are properly executed"""
```

Comment:
```suggestion
    """ensure that the crop cutting routines are properly executed"""
```
I think it would be good to add here that this method also tests for errors being raised properly.